### ISSUE TITLE
Release new version

### ISSUE THREAD

**Issue by Anteru** [pygments/pygments-migration-test#714]
(Original issue 1007 created by nobbz on 2014-06-20T10:11:46.098017+00:00)

Hi!

There was a pullrequest about a year ago that has been merged, but pygments installed via pip seems not to contain this changeset.

So when is a new pygments release planned?

The pullrequest in question is [#193](https://bitbucket.org/birkenfeld/pygments-main/pull-request/193/lexer-for-iso-iec-14977-ebnf-grammars).

**Comment by Anteru** [comment]
(Original issue 1007 last updated on 2015-10-17T02:43:24.491135+00:00)

(Issue automaticaly closed due to status in Bitbucket: resolved)

**Comment by Anteru** [comment]
(Original comment by mani95lisa on 2014-06-21T03:35:14.149066+00:00)

+1 Please release :)

**Comment by Anteru** [comment]
(Original comment by nobbz on 2014-06-24T09:44:39.612273+00:00)

4 days and no response from official project coordinators? That's a pitty...

**Comment by Anteru** [comment]
(Original comment by mani95lisa on 2014-06-24T09:48:27.827188+00:00)

Yes, I'm waiting for this too.
Hope god help us......

**Comment by Anteru** [comment]
(Original comment by nobbz on 2014-06-24T10:06:37.337625+00:00)

I just discovered that this is a duplicate of #929, which is marked as resolved,  but actually there does not seem to be a new version available.

**Comment by Anteru** [comment]
(Original comment by mani95lisa on 2014-06-24T10:24:40.646898+00:00)

Yes, I have found that, no new version is released.
I don't know how to do this, or I'll try to build :(

**Comment by Anteru** [comment]
(Original comment by nobbz on 2014-06-24T10:28:00.095173+00:00)

Building myself is out of question for me, I'm working on a project for my university where I am allowed to depend only on released versions.

**Comment by Anteru** [comment]
(Original comment by mani95lisa on 2014-06-24T10:32:13.158610+00:00)

:smile: So could you please build this for me? I can't build because I have no python. I just want to make this Node.js module work : https://github.com/rvagg/node-pygmentize-bundled  it needs python 2 and python 3 compiled version.

Thank you very much, you're my god :smile:

**Comment by Anteru** [comment]
(Original comment by nobbz on 2014-06-24T10:42:21.422766+00:00)

Thats not that easy…

1. you need python anyway to use pygments (as mentioned in the [FAQ of pygments](http://pygments.org/faq/)):
>**[What are the system requirements?](http://pygments.org/faq/#what-are-the-system-requirements)**
>
>Pygments only needs a standard Python install, version 2.6 or higher or version 3.3 or higher for Python 3. No additional libraries are needed.
1. The project you mention comes with pygments bundled, so it doesn't matter what pygments is installed on your system, the version bundled with that module will be used.

Solution to your problem:

1. Install python
2. Fork that node-module and replace the pygments in the vendor folder with a current one. (Might work, but is not guaranteed to! I do not know about neither node.js nor the internals of that package in particular)

**Comment by Anteru** [comment]
(Original comment by mani95lisa on 2014-06-24T10:46:48.401182+00:00)

@nobbz Thanks for your explain, I'll have a try later :smile:

**Comment by Anteru** [comment]
(Original comment by jaingaurav2 on 2014-06-24T14:04:22.299338+00:00)

Putting out a new release takes time, which is something that the maintainers are short on. I'm sure that they are aware of the desire to have an official release. It's just a matter of finding the time to get it done.

Please be patient and I'm sure a new release will be put out soon. You should be able to still use pygments in your project. You could download the source an possibly do something like:


```
#!python

import sys
sys.path.append('DOWNLOAD_DIRECTORY/pygments')
```

**Comment by Anteru** [comment]
(Original comment by mani95lisa on 2014-06-24T14:12:20.359104+00:00)

@coolgauravjain90 Thanks for your reply. I hope so. I'm not python developer, could you please give me any instruction about how to build 2.7 and 3.0 version?

Thanks!

**Comment by Anteru** [comment]
(Original comment by nobbz on 2014-06-24T16:40:05.881230+00:00)

I can't because I neither have a python project nor would I be allowed to
have unreleased dependencys on third party projects. So it seems that I
have to look for an alternative way to get beautiful ebnf in my
latex-document.

**Comment by Anteru** [comment]
(Original comment by dscorbett on 2014-06-25T13:12:07.450651+00:00)

@nobbz Version 0.6.0 of GitHub’s pygments.rb [contains EbnfLexer](https://github.com/tmm1/pygments.rb/blob/v0.6.0/vendor/pygments-main/pygments/lexers/text.py#L1882); could you use that?

**Comment by Anteru** [comment]
(Original comment by tshatch on 2014-10-07T05:00:20.989486+00:00)

New version coming Real Soon!

**Comment by Anteru** [comment]
(Original comment by tshatch on 2015-10-17T02:43:24.500350+00:00)

2.0 was released ages ago; follow #1154 for 2.1
